admin-ui/plugins/fido/helper/utils.ts

```diff
@@ -56,14 +66,21 @@ const transformToFormValues = (configuration?: Record<string, unknown>, type?: s
         metricReporterInterval: configuration?.metricReporterInterval || '',
         metricReporterKeepDataDays: configuration?.metricReporterKeepDataDays || '',
         personCustomObjectClassList: configuration?.personCustomObjectClassList || [],
-        hints: arrayValidationWithSchema(
-          (configuration as any)?.fido2Configuration?.hints,
-          PublicKeyCredentialHints,
-        ),
+        hints: arrayValidationWithSchema(fido2Config?.hints || [], PublicKeyCredentialHints),
       }
 }
 
-const createFidoConfigPayload = ({ fidoConfiguration, data, type }: any) => {
+interface CreateFidoConfigPayloadParams {
+  fidoConfiguration: { fido: AppConfiguration }
+  data: Record<string, unknown>
+  type: string
+}
+
+const createFidoConfigPayload = ({
+  fidoConfiguration,
+  data,
+  type,
+}: CreateFidoConfigPayloadParams) => {
   const payload = JSON.parse(JSON.stringify(fidoConfiguration.fido))
   if (type === fidoConstants.STATIC) {
```

Comment:
Avoid JSON deep clone and preserve types

`JSON.parse(JSON.stringify(...))` drops types and can mutate numbers/undefined. Prefer a typed shallow clone of the object and its nested fido2Configuration.

Apply:

```diff
-const createFidoConfigPayload = ({
+const createFidoConfigPayload = ({
   fidoConfiguration,
   data,
   type,
 }: CreateFidoConfigPayloadParams) => {
-  const payload = JSON.parse(JSON.stringify(fidoConfiguration.fido))
+  const base = fidoConfiguration.fido as AppConfiguration
+  const payload: AppConfiguration = {
+    ...base,
+    fido2Configuration: base.fido2Configuration
+      ? { ...base.fido2Configuration }
+      : undefined,
+  }
```